lms/djangoapps/experiments/tests/test_utils.py

```diff
@@ -116,3 +121,38 @@ def test_price_and_sku_from_multiple_courses(self):
         self.assertEqual(2, len(skus))
         self.assertIn(self.run_a_sku, skus)
         self.assertIn(self.entitlement_a_sku, skus)
+
+    def test_get_experiment_user_metadata_context(self):
+        course = CourseFactory.create(start=now() - timedelta(days=30), pacing_type="instructor_paced", course_duration=None, upgrade_price='Free',
```

Comment:
What happens if it doesn't successfully get the course information, or something else is missing that is required when requesting the user metadata?

Reply:
Just saw the later convo with JJ down the line 👍

Reply:
Yes this is added now- thanks for raising!